index.d.ts

```diff
@@ -1553,6 +1553,16 @@ declare namespace Dysnomia {
     default_values?: SelectMenuDefaultValue<"channel">[];
     type: Constants["ComponentTypes"]["CHANNEL_SELECT"];
   }
+  interface ContainerComponent extends ComponentBase {
+    type: Constants["ComponentTypes"]["CONTAINER"];
+    accent_color?: number;
+    spoiler?: boolean;
+    components: (ActionRow | TextDisplayComponent | SectionComponent | MediaGalleryComponent | SeparatorComponent | FileComponent);
```

Comment:
don't have access to the spec, but this should probably be an array?
```suggestion
    components: (ActionRow | TextDisplayComponent | SectionComponent | MediaGalleryComponent | SeparatorComponent | FileComponent)[];
```

Reply:
Yep, forgot about that sorry.